src/mono/wasm/runtime/net6-legacy/js-to-cs.ts

```diff
@@ -192,8 +186,7 @@ export function js_typed_array_to_array(js_obj: any): MonoArray {
     }
 }
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/explicit-module-boundary-types
-export function js_to_mono_enum(js_obj: any, method: MonoMethod, parmIdx: number): number {
+export function js_to_mono_enum(js_obj: any): number {
```

Comment:
this is fine in this case, but be aware that changing the signature of a function to remove unused args can have observable impact on behavior since it changes things like Function.length and Function.toString

Reply:
An argument count mismatch also causes deoptimization in js runtimes in some cases, but I don't think this is hot code